README.md

```diff
@@ -1,5 +1,206 @@
+    * [Joystick](landside/catkin_ws/src/joystick) - Allows manual joystick control for testing.
+    * [Simulation](simulation) - Physics-enabled simulation that can be used for local testing. (not in landside, but run using the landside container)
+
+
+## Flow 
+
+The general flow of information between components is shown in the diagram below:
+
+(Diagram to be added)
+
+
+## Running Our Docker Images
+
+### Required Software
+1. Download and install the appropriate Docker client.
+
+    * [Mac](https://docs.docker.com/docker-for-mac/install/)
+    * [Windows (Home)](https://docs.docker.com/docker-for-windows/install-windows-home/)
+    * [Windows (Pro, Education, etc.)](https://docs.docker.com/docker-for-windows/install/)<br>
+        :information_source: Which Windows do I have? *Settings > System > About > look at "Edition"*
+
+1. If you want graphics forwarding over SSH:
+
+    * [Mac: XQuartz](https://www.xquartz.org/)
+    * [Windows: MobaXterm](https://mobaxterm.mobatek.net/)<br>
+        :warning: Open MobaXterm > click on Settings > go the X11 tab > uncheck RANDR. This allows us to use [rviz](http://wiki.ros.org/rviz) over SSH.
+
+### Pool Testing
+Use these instructions when running code on the robot itself.
+
+1. Connect your computer to the robot via Ethernet. Then SSH into the robot's onboard computer.
+    ```bash
+    ssh robot@192.168.1.1
+    ```
+1. There, make sure the latest code from this repo is pulled. Then, run the onboard container if it's not already running.
+    ```bash
+    docker run -td --privileged --net=host -v /home/robot/robosub-ros:/root/dev/robosub-ros dukerobotics/robosub-ros:onboard
+    ```
+1. Clone this repo on your local computer. In the newly-created directory (robosub-ros), run the landside container. If on Windows, use PowerShell.
+    ```bash
+    docker run -td -p 2201:2201 -v ${PWD}:/root/dev/robosub-ros dukerobotics/robosub-ros:landside
+    ```
+1. SSH into the onboard container. Password is `robotics`.
+    ```bash
+    ssh -p 2200 root@192.168.1.1
+    ```
+1. SSH into the landside container. Password is `robotics`.
+    ```bash
+    ssh -XY -p 2201 root@localhost
+    ```
+1. Inside landside, complete the connection between the two containers.
+    ```bash
+    source /opt/ros/melodic/setup_network.bash
+    ```
+
+### Local Testing
+Use these instructions to test code on your computer by simulating the robot's execution.
+
+1. To run the containers, clone this repo. In the newly-created directory (`robosub-ros`), execute
+    ```bash
+    docker-compose up -d
+    ```
+    This will pull the images if you don't have them, create a new network that simulates the network we use on our robot, mount the code, and start the containers.
+    
+    To update the images, or to just pull them without running them, use `docker-compose pull`.
+
+1. SSH into the onboard container. Password is `robotics`.
```

Comment:
numbering, for this and following steps

Reply:
see previous comment on numbering. tl;dr we'll keep it as is.